src/action-creators/deleteThought.js

```diff
@@ -83,13 +82,13 @@ export const deleteThought = () => {
 
   // must call store.getState() to use the new state after existingThoughtDelete
   const next = perma(() => showContexts
-    ? unroot(getContextsSortedAndRanked(store.getState(), headValue(contextOf(path))))[0]
+    ? unroot(getContextsSortedAndRanked(state, headValue(contextOf(path))))[0]
     // get first visible thought
-    : (sortPreference === 'Alphabetical' ? getThoughtsSorted : getThoughtsRanked)(store.getState(), context)
+    : (sortPreference === 'Alphabetical' ? getThoughtsSorted : getThoughtsRanked)(state, context)
       .find(isVisible)
```

Comment:
Note the comment above. Please revert back to `store.getState()`, otherwise the state will be outdated by the time `next()` is called.

Reply:
Ahh I understand. I will revert it.